### PR TITLE
Cpufreq code style

### PR DESCRIPTION
**Pull request by raveit65**
- tabs to spaces
- 4 spaces at the beginning of a line

### CODE REVIEW

File: cpufreq/src/cpufreq-monitor.c [hunk 1]
```diff
@@ -186,61 +186,61 @@ cpufreq_monitor_set_property (GObject      *object,
...
-                break;
-        case PROP_MAX_FREQUENCY: {
-                gint freq = g_value_get_int (value);
-
-                if (freq != monitor->priv->max_freq) {
-                        monitor->priv->max_freq = freq;
-                        monitor->priv->changed = TRUE;
-                }
+    }
+        break;
```
Comment (rbuj) [inline cpufreq/src/cpufreq-monitor.c:215]: break sentence should be within case scope

See AfterCaseLabel in https://clang.llvm.org/docs/ClangFormatStyleOptions.html

Captura de Pantalla 2020-11-19 a les 22 17 07

File: cpufreq/src/cpufreq-monitor.c [hunk 2]
```diff
@@ -249,150 +249,150 @@ cpufreq_monitor_get_property (GObject    *object,
...
+        return NULL;
+
+    if (monitor->priv->available_freqs)
         return monitor->priv->available_freqs;
+
+    class = CPUFREQ_MONITOR_GET_CLASS (monitor);
+
+    if (class->get_available_frequencies) {
+        monitor->priv->available_freqs = class->get_available_frequencies (monitor);
+    }
```
Comment (rbuj) [inline cpufreq/src/cpufreq-monitor.c:328]: If there is only one line in the if-body it's not necessary to use the curly brackets

File: cpufreq/src/cpufreq-prefs.c [hunk 3]
```diff
@@ -31,554 +31,564 @@
...
-			      NULL);
+    gint cpu;
+
+    cpu = gtk_combo_box_get_active (GTK_COMBO_BOX (prefs->priv->cpu_combo));
+
+    if (cpu >= 0) {
+        g_object_set (G_OBJECT (prefs),
+                      "cpu", cpu,
+                      NULL);
         }
```
Comment (rbuj) [inline cpufreq/src/cpufreq-prefs.c:335]: The curly bracket '}' is not aligned with the keyword if

File: cpufreq/src/cpufreq-selector.c [hunk 4]
```diff
@@ -70,257 +70,257 @@ cpufreq_selector_init (CPUFreqSelector *selector)
...
 cpufreq_selector_set_frequency_async (CPUFreqSelector *selector,
-				      guint            cpu,
-				      guint            frequency)
+                                      guint            cpu,
+                                      guint            frequency)
 {
     guint            cores;
     cores = get_nprocs() ;
 
     for (cpu = 0; cpu < cores; cpu = cpu+1){
```
Comment (rbuj) [inline cpufreq/src/cpufreq-selector.c:302]: missing a space `...cpu = cpu+1) {`

File: cpufreq/src/cpufreq-selector/cpufreq-selector-sysfs.c [hunk 5]
```diff
@@ -27,18 +27,22 @@
...
 struct _CPUFreqSelectorSysfsPrivate {
-	GList *available_freqs;
-	GList *available_govs;
+    GList *available_freqs;
+    GList *available_govs;
 };
 
-static void     cpufreq_selector_sysfs_finalize      (GObject                   *object);
+static void
+cpufreq_selector_sysfs_finalize (GObject                   *object);
```
Comment (rbuj) [inline cpufreq/src/cpufreq-selector/cpufreq-selector-sysfs.c:35]: cpufreq_selector_sysfs_finalize columns are not aligned with cpufreq_selector_sysfs_set_frequency, cpufreq_selector_sysfs_set_governor

File: cpufreq/src/cpufreq-selector/cpufreq-selector.c [hunk 6]
```diff
@@ -23,143 +23,145 @@
...
-        }
+    CPUFreqSelector *selector = CPUFREQ_SELECTOR (object);
+
+    switch (prop_id) {
+    case PROP_CPU:
+        selector->priv->cpu = g_value_get_uint (value);
+        break;
+    default:
+        G_OBJECT_WARN_INVALID_PROPERTY_ID (object, prop_id, spec);
+        break;
```
Comment (rbuj) [inline cpufreq/src/cpufreq-selector/cpufreq-selector.c:108]: default doesn't require the break statement

File: cpufreq/src/cpufreq-monitor.c (hunk 1 above)
Comment (raveit65) [reply cpufreq/src/cpufreq-monitor.c:215]: Like this?
```
    switch (prop_id) {
    case PROP_CPU: {
        guint cpu = g_value_get_uint (value);

        if (cpu != monitor->priv->cpu) {
            monitor->priv->cpu = cpu;
            monitor->priv->changed = TRUE;
        }
        break;
    }
    case PROP_ONLINE:
        monitor->priv->online = g_value_get_boolean (value);

        break;
    case PROP_FREQUENCY: {
        gint freq = g_value_get_int (value);

        if (freq != monitor->priv->cur_freq) {
            monitor->priv->cur_freq = freq;
            monitor->priv->changed = TRUE;
        }
        break;
    }
    case PROP_MAX_FREQUENCY: {
        gint freq = g_value_get_int (value);

        if (freq != monitor->priv->max_freq) {
            monitor->priv->max_freq = freq;
            monitor->priv->changed = TRUE;
        }
        break;
    }
    case PROP_GOVERNOR: {
        const gchar *gov = g_value_get_string (value);

        if (monitor->priv->governor) {
            if (g_ascii_strcasecmp (gov, monitor->priv->governor) != 0) {
                g_free (monitor->priv->governor);
                monitor->priv->governor = gov ? g_strdup (gov) : NULL;
                monitor->priv->changed = TRUE;
            }
        } else {
            monitor->priv->governor = gov ? g_strdup (gov) : NULL;
            monitor->priv->changed = TRUE;
        }
        break;
    }
    default:
                G_OBJECT_WARN_INVALID_PROPERTY_ID (object, prop_id, spec);
        break;
    }
```
Then it needs to be fixed at other places too.

Comment (raveit65) [reply cpufreq/src/cpufreq-monitor.c:215]: Oh, the break after default can be removed.

File: cpufreq/src/cpufreq-selector/cpufreq-selector.c (hunk 6 above)
Comment (raveit65) [reply cpufreq/src/cpufreq-selector/cpufreq-selector.c]: Fixed here and at line 125.

File: cpufreq/src/cpufreq-monitor.c (hunk 1 above)
Comment (rbuj) [reply cpufreq/src/cpufreq-monitor.c:215]: Yes. The curly brackets are required in a case-block to delimit the scope of the declared variables within the block, by doing it this way, it makes easier to read the switch body, as the separation of cases is visually clearer.

File: cpufreq/src/cpufreq-prefs.c [hunk 7]
```diff
@@ -31,554 +31,564 @@
...
+                                      TRUE);
+
+        break;
+    case CPUFREQ_MODE_TEXT_PERCENTAGE:
+        gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (prefs->priv->show_perc),
+                                      TRUE);
+
+        break;
+    }
 }
```
Comment (raveit65) [inline cpufreq/src/cpufreq-prefs.c:447]: @rbuj 
Is a `default` missing inthe case scope?

File: cpufreq/src/cpufreq-monitor.c (hunk 1 above)
Comment (raveit65) [reply cpufreq/src/cpufreq-monitor.c:214]: Done.
I just updated `cpufreq-prefs.c`(pt.7), Can you please take a look again?

File: cpufreq/src/cpufreq-prefs.c (hunk 7 above)
Comment (rbuj) [reply cpufreq/src/cpufreq-prefs.c:447]:
```C
/* gcc -Wall switch.c -o switch */
#include <stdio.h>

void test (char option)
{
   if (option == 'A') {
        printf ("<%c>", option);
	goto end;
   }

   switch (option) {
        case 'a':
        case 'b':
                printf ("a or b?, %c\n", option);
        case 'c':
                printf ("a, b or c? %c", option);
		break;
        case 'd': {
		int i;
		for (i=0;i<9;i++)
	                printf ("%d", i);
		printf (" %c", option);
                break;
        }
	case 'e':
		printf ("%c", option);
        default:
               printf ("?");
               /* goto end; */
   }
end:
   printf ("\n---\n");
}

int main (void)
{
   test ('a');
   test ('b');
   test ('c');
   test ('d');
   test ('e');
   test ('z');
   test ('A');

   return 0;
}
```
```
$ ./switch 
a or b?, a
a, b or c? a
---
a or b?, b
a, b or c? b
---
a, b or c? c
---
012345678 d
---
e?
---
?
---
<A>
---
```

Comment (rbuj) [reply cpufreq/src/cpufreq-prefs.c:447]: You can use `g_assert_not_reached ()` in order to check unexpected value of `prefs->priv->show_text_mode`:
```C
default:
    g_assert_not_reached ();
```

**Comment by rbuj** [comment]
[file.patch.gz](https://github.com/mate-desktop/mate-applets/files/5573593/file.patch.gz)

Remove trailing whitespaces spaces:
```shell
$ find . -type f -name '*.c' -exec sed --in-place 's/[[:space:]]\+$//' {} \+
```
+fix 1 indent:
```diff
diff --git a/cpufreq/src/cpufreq-monitor-sysfs.c b/cpufreq/src/cpufreq-monitor-sysfs.c
index 1e123bbe..0d7f378a 100644
--- a/cpufreq/src/cpufreq-monitor-sysfs.c
+++ b/cpufreq/src/cpufreq-monitor-sysfs.c
@@ -268,7 +268,7 @@ cpufreq_monitor_sysfs_get_available_frequencies (CPUFreqMonitor *monitor)
     g_object_get (G_OBJECT (monitor),
                   "cpu", &cpu, NULL);
 
-    path = g_strdup_printf (CPUFREQ_SYSFS_BASE_PATH, cpu, 
+    path = g_strdup_printf (CPUFREQ_SYSFS_BASE_PATH, cpu,
                             "scaling_available_frequencies");
 
     if (!cpufreq_file_get_contents (path, &buffer, NULL, &error)) {
diff --git a/cpufreq/src/cpufreq-monitor.c b/cpufreq/src/cpufreq-monitor.c
index d55b2166..a0a2510f 100644
--- a/cpufreq/src/cpufreq-monitor.c
+++ b/cpufreq/src/cpufreq-monitor.c
@@ -99,9 +99,9 @@ cpufreq_monitor_class_init (CPUFreqMonitorClass *klass)
                                      g_param_spec_uint ("cpu",
                                                         "CPU",
                                                         "The cpu to monitor",
-                                                        0, 
+                                                        0,
                                                         G_MAXUINT,
-                                                        0, 
+                                                        0,
                                                         G_PARAM_CONSTRUCT |
                                                         G_PARAM_READWRITE));
     g_object_class_install_property (object_class,
@@ -116,7 +116,7 @@ cpufreq_monitor_class_init (CPUFreqMonitorClass *klass)
                                      g_param_spec_int ("frequency",
                                                        "Frequency",
                                                        "The current cpu frequency",
-                                                       0, 
+                                                       0,
                                                        G_MAXINT,
                                                        0,
                                                        G_PARAM_READWRITE));
@@ -238,7 +238,7 @@ cpufreq_monitor_set_property (GObject      *object,
         break;
     }
     default:
-                G_OBJECT_WARN_INVALID_PROPERTY_ID (object, prop_id, spec);
+        G_OBJECT_WARN_INVALID_PROPERTY_ID (object, prop_id, spec);
     }
 }
 
diff --git a/cpufreq/src/cpufreq-popup.c b/cpufreq/src/cpufreq-popup.c
index c87bfffa..56c1b0d0 100644
--- a/cpufreq/src/cpufreq-popup.c
+++ b/cpufreq/src/cpufreq-popup.c
@@ -285,7 +285,7 @@ frequencies_menu_create_actions (CPUFreqPopup *popup)
         g_free (unit);
 
         cpufreq_popup_menu_add_action (popup,
-                                       "Frequency", 
+                                       "Frequency",
                                        popup->priv->freqs_group,
                                        text, label, TRUE);
         g_free (label);
diff --git a/cpufreq/src/cpufreq-prefs.c b/cpufreq/src/cpufreq-prefs.c
index a0ef49be..5f4ab01d 100644
--- a/cpufreq/src/cpufreq-prefs.c
+++ b/cpufreq/src/cpufreq-prefs.c
@@ -309,7 +309,7 @@ cpufreq_prefs_dialog_show_perc_toggled (GtkWidget    *show_perc,
 {
 
     CPUFreqShowTextMode show_text_mode;
-    
+
     if (gtk_toggle_button_get_active (GTK_TOGGLE_BUTTON (show_perc))) {
         /* Show cpu usage in percentage */
         show_text_mode = CPUFREQ_MODE_TEXT_PERCENTAGE;
@@ -572,7 +572,7 @@ cpufreq_prefs_dialog_create (CPUFreqPrefs *prefs)
                       (gpointer) prefs);
 }
 
-void 
+void
 cpufreq_preferences_dialog_run (CPUFreqPrefs *prefs, GdkScreen *screen)
 {
     g_return_if_fail (CPUFREQ_IS_PREFS (prefs));
diff --git a/cpufreq/src/cpufreq-selector.c b/cpufreq/src/cpufreq-selector.c
index b89512fa..9bd63f93 100644
--- a/cpufreq/src/cpufreq-selector.c
+++ b/cpufreq/src/cpufreq-selector.c
@@ -196,7 +196,7 @@ cpufreq_selector_set_frequency_async (CPUFreqSelector *selector,
 {
     guint            cores;
     cores = get_nprocs() ;
-    for (cpu = 0; cpu < cores; cpu = cpu+1){ 
+    for (cpu = 0; cpu < cores; cpu = cpu+1){
         SelectorAsyncData *data;
 
         data = g_new0 (SelectorAsyncData, 1);
diff --git a/cpufreq/src/cpufreq-selector/cpufreq-selector-libcpufreq.c b/cpufreq/src/cpufreq-selector/cpufreq-selector-libcpufreq.c
index 9ce8e4f6..7167eb55 100644
--- a/cpufreq/src/cpufreq-selector/cpufreq-selector-libcpufreq.c
+++ b/cpufreq/src/cpufreq-selector/cpufreq-selector-libcpufreq.c
@@ -74,13 +74,13 @@ cpufreq_selector_libcpufreq_new (guint cpu)
     CPUFreqSelector *selector;
 
     selector = CPUFREQ_SELECTOR (g_object_new (CPUFREQ_TYPE_SELECTOR_LIBCPUFREQ,
-                                               "cpu", cpu, 
+                                               "cpu", cpu,
                                                NULL));
 
     return selector;
 }
 
-static guint 
+static guint
 cpufreq_selector_libcpufreq_get_valid_frequency (CPUFreqSelectorLibcpufreq *selector,
                                                  guint                      frequency)
 {
@@ -136,7 +136,7 @@ cpufreq_selector_libcpufreq_set_frequency (CPUFreqSelector *selector,
                      SELECTOR_ERROR_SET_FREQUENCY,
                      "Cannot set frequency '%d'",
                      frequency);
-                
+
         return FALSE;
     }
 
diff --git a/cpufreq/src/cpufreq-selector/cpufreq-selector-procfs.c b/cpufreq/src/cpufreq-selector/cpufreq-selector-procfs.c
index a69caa03..002c387c 100644
--- a/cpufreq/src/cpufreq-selector/cpufreq-selector-procfs.c
+++ b/cpufreq/src/cpufreq-selector/cpufreq-selector-procfs.c
@@ -153,7 +153,7 @@ cpufreq_procfs_write (const gchar *path,
     return TRUE;
 }
 
-static gboolean 
+static gboolean
 cpufreq_selector_procfs_set_frequency (CPUFreqSelector *selector,
                                        guint            frequency,
                                        GError         **error)
@@ -205,7 +205,7 @@ cpufreq_selector_procfs_set_frequency (CPUFreqSelector *selector,
 
     return TRUE;
 }
-        
+
 static gboolean
 cpufreq_selector_procfs_set_governor (CPUFreqSelector *selector,
                                       const gchar     *governor,
diff --git a/cpufreq/src/cpufreq-selector/cpufreq-selector-sysfs.c b/cpufreq/src/cpufreq-selector/cpufreq-selector-sysfs.c
index 8f433668..b3dff154 100644
--- a/cpufreq/src/cpufreq-selector/cpufreq-selector-sysfs.c
+++ b/cpufreq/src/cpufreq-selector/cpufreq-selector-sysfs.c
@@ -93,7 +93,7 @@ cpufreq_selector_sysfs_new (guint cpu)
     CPUFreqSelector *selector;
 
     selector = CPUFREQ_SELECTOR (g_object_new (CPUFREQ_TYPE_SELECTOR_SYSFS,
-                                               "cpu", cpu, 
+                                               "cpu", cpu,
                                                NULL));
 
     return selector;
@@ -183,7 +183,7 @@ cpufreq_selector_sysfs_get_freqs (CPUFreqSelectorSysfs *selector)
                             "cpu", &cpu,
                             NULL);
 
-    path = g_strdup_printf (CPUFREQ_SYSFS_BASE_PATH, cpu, 
+    path = g_strdup_printf (CPUFREQ_SYSFS_BASE_PATH, cpu,
                             "scaling_available_frequencies");
 
     buffer = cpufreq_sysfs_read (path, &error);
@@ -329,7 +329,7 @@ cpufreq_selector_sysfs_get_govs (CPUFreqSelectorSysfs *selector)
                   "cpu", &cpu,
                   NULL);
 
-    path = g_strdup_printf (CPUFREQ_SYSFS_BASE_PATH, cpu, 
+    path = g_strdup_printf (CPUFREQ_SYSFS_BASE_PATH, cpu,
                             "scaling_available_governors");
 
     buffer = cpufreq_sysfs_read (path, &error);
diff --git a/cpufreq/src/cpufreq-selector/main.c b/cpufreq/src/cpufreq-selector/main.c
index c537be8a..749653db 100644
--- a/cpufreq/src/cpufreq-selector/main.c
+++ b/cpufreq/src/cpufreq-selector/main.c
@@ -192,7 +192,7 @@ main (gint argc, gchar **argv)
         if (error) {
             g_printerr ("%s\n", error->message);
             g_error_free (error);
-        } 
+        }
 
         g_option_context_free (context);
 
diff --git a/cpufreq/src/cpufreq-utils.c b/cpufreq/src/cpufreq-utils.c
index 03ad01eb..a49d27c6 100644
--- a/cpufreq/src/cpufreq-utils.c
+++ b/cpufreq/src/cpufreq-utils.c
@@ -292,7 +292,7 @@ cpufreq_file_get_contents (const gchar *filename,
         if (bytes_read < 0) { /* Error */
             if (errno != EINTR) {
                 gint save_errno = errno;
-                                
+
                 g_set_error (error,
                              G_FILE_ERROR,
                              g_file_error_from_errno (save_errno),
@@ -305,7 +305,7 @@ cpufreq_file_get_contents (const gchar *filename,
 
                 g_free (display_filename);
                 close (fd);
-                                
+
                 return FALSE;
             }
         }
```

**Review by rbuj — APPROVED** [review]
LGTM

**Comment by raveit65** [comment]
Thanks for the review.
I will merge it until travis builds are finished.
Btw. Travis move all workers from travis-ci.org to travis-ci.com, an slowing down builds is a side effect.
We need to migrate to, but there is a new pricing model on travis-ci.com.
But Builds are limited to 100000 credits (1000 build min) on free plan.
This isn't enough for us........
Well, i wrote them a mail and ask for a special free plan for open-source project like us.
More infos at
https://travis-ci.community/t/builds-hang-in-queued-state/10250/5
https://docs.travis-ci.com/user/migrate/open-source-repository-migration#frequently-asked-questions
https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing

**Comment by raveit65** [comment]
Opps, you merged it already :)

**Comment by rbuj** [comment]
ok, thank you for the info.The continuous integration is fine, and I take this into account when I do a version, however I usually test all the requests locally as I always do runtime tests. I don't do all the tests that are done in ci, I make other tests based on the code to review.

**Comment by selectiveduplicate** [comment]
We might start considering GitHub Actions, given the situation with Travis.

**Pull request merged**